dev/Dockerfile

```diff
@@ -10,74 +10,130 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-FROM debian:jessie
+# Base layer containing dependencies needed at runtime. This layer will be
+# cached after the initial build.
+FROM debian:jessie AS runtime
 
-MAINTAINER CouchDB Developers dev@couchdb.apache.org
+LABEL maintainer="CouchDB Developers <dev@couchdb.apache.org>"
```

Comment:
Is this the new format? If so, can you issue a separate PR to improve this in the other Dockerfiles? Thanks! :)

Reply:
Yes, MAINTAINER is listed as deprecated in the Dockerfile docs:

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Will file another PR, thanks.